src/views/auth/error.html

```diff
@@ -0,0 +1,9 @@
+{% extends 'default.html' %}
+{% set title = 'Authentication Error' %}
+{% block pageHeading %}Oops!{% endblock %}
```

Comment:
Perhaps for the `TOO_YOUNG` error, we could have a title like "We're sorry" instead?